docs/source/using/profiles.adoc

```diff
@@ -63,75 +69,66 @@ The default profile is *minishift*.
 It will be present by default and it does not need to be created.
 ====
 
-[[starting-a-profile]]
-== Starting A Profile
-
-You can start the active profile with xref:../command-ref/minishift_start.adoc#[`minishift start`] command.
-To start a non active profile, you need to use the `--profile` flag.
-
-To start a non active profile, run:
-
-----
-$ minishift --profile PROFILE_NAME start
-----
-
-NOTE: A profile is automatically set to active profile when the VM instance is started successfully.
-
 [[listing-profiles]]
 == Listing Profiles
 
 You can list all existing profiles with the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command.
-You can also see the active profile is highlighted in the output.
-
-To list profiles, run:
+You can also see the active profile highlighted in the output.
 
 ----
 $ minishift profile list
+- minishift     Running     	(Active)
+- profile-demo  Does Not Exist
 ----
 
 [[switching-profiles]]
 == Switching Profiles
 
-You can switch between profiles with the xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] command.
-Only one profile can be active at any time.
-
-To switch between profiles, run:
+To switch between profiles use the xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] command:
 
 ----
-$ minishift profile set PROFILE_NAME
+$ minishift profile set profile-demo
+Profile 'profile-demo' set as active profile
 ----
 
+NOTE: Only one profile can be active at any time.
+
 [[deleting-profiles]]
 == Deleting Profiles
 
 To delete a profile, run:
 
 ----
-$ minishift profile delete PROFILE_NAME
+$ minishift profile delete profile-demo
+You are deleting the active profile. It will remove the VM and all related artifacts. Do you want to continue [y/N]?: y
+Deleted:  /Users/john/.minishift/profiles/profile-demo
+Profile 'profile-demo' deleted successfully
+Switching to default profile 'minishift' as the active profile.
 ----
 
 NOTE: The default profile *minishift* cannot be deleted.
 
 [[example-workflows-profiles]]
-== Example Workflows
+== Example Workflow For Profile Configuration
 
-To create a new profile with specific configurations, run:
+To create a new profile and configure its the xref:../using/basic-usage.adoc#persistent-configuration[persistent configuration] you have two options.
```

Comment:
Typo: "its the"

I'd also move the clause "you have two options" to the beginning of the sentence:

"You have two options to create a new profile and configure its...."

Reply:
+1 for re-arranging